### DISCUSSION TITLE
External Storage Unmounted from Host

### DISCUSSION THREAD

**Discussion by richardcrbb** [nextcloud/all-in-one#5794] [Questions] [answered]
Hi there, I use a RPI5 with docker and Operating System: Linux 6.8.0-1017-raspi aarch64

CPU: Raspberry Pi 5 Model B Rev 1.0 (4 cores)

Memory: 7.75 GB

Server time: Thu Dec 26 02:23:22 UTC 2024 

sda1
Mount: /mnt/nube
Filesystem: ext4
Size: 1.70 TB
Available: 1.66 TB
Used: 3% (41.43 GB)

Nextcloud AIO v9.3.0

I unmounted the SDA1 from my ubuntu server and opened the nextcloud ui and tried to access the directory Photos in my external storage and couldn´t of course, but when I mounted my SDA1 again and reboot the server, I don´t see the Photos directory in my external storage anymore....  I was worried I may have deleted the directory so I looked for it through ubuntu bash and the directory is indeed in the SDA1 still.

How can I sync my external storage ?????? without losing the configuration of the shared directories, I don;t want to delete and add again this external storage.

There is no log in the system about this error

**Comment by szaimen** [comment, marked as answer]
Hi, please ask this at https://help.nextcloud.com. Thanks!
